corehq/apps/app_manager/static/app_manager/js/details/utils.js

```diff
@@ -45,8 +45,15 @@ hqDefine("app_manager/js/details/utils", function () {
 
         if (hqImport('hqwebapp/js/toggles').toggleEnabled('CASE_LIST_MAP')) {
             formats.push({
-            value: "address-popup",
-            label: gettext('Address Popup (Web Apps only)'),
+                value: "address-popup",
+                label: gettext('Address Popup (Web Apps only)'),
+            });
+        }
+
+        if (hqImport('hqwebapp/js/toggles').toggleEnabled('CASE_LIST_CLICKABLE_ICON')) {
+            formats.push({
+                value: "clickable-icon",
+                label: gettext('Clickable Icon (Web Apps only)'),
```

Comment:
Just wondering about this name. Might be good to check with product what to call it in the UI.

Reply:
I see you changed that in your PR. So I think we are good here.

Reply:
Did I? I don't think I did.